File: src/main/java/model/Book.java
package model;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

import java.io.Serializable;

@DatabaseTable(tableName = "books")
public class Book extends Model implements Serializable{

    @SuppressWarnings("unused")
    @DatabaseField
    private String title;

    @SuppressWarnings("unused")
    @DatabaseField
    private String author;

    @SuppressWarnings("unused")
    @DatabaseField
    private String other;
}

File: src/main/kotlin/model/Book.kt
package model

import java.io.Serializable

data class Book(
        var title: String?,
        var author: String?,
        var other: String?,
        var image: String?
) : Model(), Serializable {
    override fun save() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }
}
